src/pynwb/tests/test_api_functions.py

```diff
@@ -2,116 +2,212 @@
 
 import numpy as np
 import pytest
+
 from ndx_microscopy.testing import (
     mock_PlanarImagingSpace,
-    mock_MicroscopyPlaneSegmentation,
+    mock_VolumetricImagingSpace,
+    mock_Segmentation2D,
+    mock_Segmentation3D,
+    mock_SegmentationContainer,
+    mock_Segmentation,
+)
+from ndx_microscopy import (
+    Segmentation2D,
+    Segmentation3D,
+    Segmentation,
 )
-from ndx_microscopy import MicroscopyPlaneSegmentation
 
 
-def test_add_roi_with_pixel_mask():
-    """Test adding ROI with pixel mask."""
+def test_planar_pixel_to_image_conversion():
```

Comment:
pixel_to_image needs a test for the ValueError case.

Ditto for each of these api functions that can throw a valueError.